go/bundle/bundle.go

```diff
@@ -836,25 +896,31 @@ func (b *Bundle) WriteTo(w io.Writer) (int64, error) {
 		return cw.Written, err
 	}
 
-	sos := []sectionOffset{
-		sectionOffset{"index", uint64(is.Len())},
-		sectionOffset{"responses", uint64(rs.Len())},
+	ms, err := newManifestSection(b.ManifestURL)
```

Comment:
Can we make Manifest section optional?

Reply:
It's optional. If `b.ManifestURL` is nil, `newManifestSection` returns nil and the manifest section won't be emitted.

Reply:
Thanks for explanation. I think we want to make that clear from reading only this part of the code.
`newManifestSection` returning `nil, nil` is a surprising behavior to me.

Some ideas:
- `if url != ""` before newManifestSection
- Alternate namings? `tryCreateManifestSection`?
- Add comment explaining the behavior here?

Reply:
Thanks for the suggestions. Chenged to check `b.ManifestURL != nil` before newManifestSection.